hooks/009-locale-archive.chroot

```diff
@@ -23,4 +23,12 @@ for encoding in "${!found_encodings[@]}"; do
     localedef --no-archive -f "${encoding}" -i C "C.${encoding_canonical}"
 done
 
+# Backup configuration
+cp /etc/locale.conf /etc/locale.conf.bak
+
+# purge
 apt purge -y locales
+
+# and restore so the default locale can be found
+mv /etc/locale.conf.bak /etc/locale.conf
```

Comment:
Could we just put LANG=C.UTF-8, without taking it from the image=

Reply:
Also it needs to be writable in theory. So we should add it to the writable-paths

Reply:
I've changed this now